CONTRIBUTING.md

```diff
@@ -192,7 +192,12 @@ that’s a sign that you probably need to split up your patch.
 
 If the patch fixes a logged bug entry, refer to that bug entry by number and
 URL. If the patch follows from a mailing list discussion, give a URL to the
-mailing list archive.
+mailing list archive. Please format these lines as `Fixes:` followed by the URL
+or, for Github bugs, the bug number preceded by a #. For example:
+
+```
+Fixes: #23480
+```
```

Comment:
Maybe add some more examples here, i.e. JIRA, bugzilla (still in use for fedora bugs).

Reply:
agree, please add